pkgs/applications/audio/goattracker/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv
+, fetchurl
+, unzip
+, makeDesktopItem
+, imagemagick
+, SDL
+}:
+
+let desktopItem = makeDesktopItem {
+  type = "Application";
+  name = "goattracker";
+  desktopName = "GoatTracker 2";
+  genericName = "Music Tracker";
+  exec = "goattrk2";
+  icon = "goattracker";
+  categories = "Multimedia;";
+};
+
+in stdenv.mkDerivation rec {
+  pname = "goattracker";
+  version = "2.75";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/goattracker2/GoatTracker%202/${version}/GoatTracker_${version}.zip";
+    sha256 = "1km97nl7qvk6qc5l5j69wncbm76hf86j47sgzgr968423g0bxxlk";
+  };
+  sourceRoot = "goattrk2/src";
+
+  nativeBuildInputs = [ unzip imagemagick ];
+  buildInputs = [ SDL ];
+
+  # PREFIX gets treated as BINDIR.
+  makeFlags = [ "PREFIX=$(out)/bin/" ];
+
+  # The zip contains some build artifacts.
+  prePatch = "make clean";
+
+  # The destination does not get created automatically.
+  preBuild = "mkdir -p $out/bin";
+
+  # Other files get installed during the build phase.
+  installPhase = ''
+    convert goattrk2.bmp goattracker.png
+    install -Dm644 goattracker.png $out/share/icons/hicolor/32x32/apps/goattracker.png
+    ${desktopItem.buildCommand}
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A crossplatform music editor for creating Commodore 64 music. Uses reSID library by Dag Lem and supports alternatively HardSID & CatWeasel devices";
+    homepage = "https://cadaver.github.io/tools.html";
+    downloadPage = "https://sourceforge.net/projects/goattracker2/";
+    license = licenses.gpl2Plus;
```

Comment:
Oh. Oh no. The licensing is awful, there are at least 3 different ones and the expr.h one is definitely not free...

Still, debian, ubuntu, a bunch of bsds, and others, are all packaging it as gpl2+ since a long time, so I myself am going to trust them and assume this is some sort of weird dual licensing.